lib/rubocop/cop/rails/hash_key_value_route.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # Avoid Hash key-value routing.
```

Comment:
Let's add the deprecation from 7.2 as a rational for this cop? This is very sparse.

Reply:
Thanks, I've added a little more explanation based on the Rails' deprecation message.

Reply:
That's good, thanks! And my bad, it'll only be deprecated in 8.0 (removed in 8.1) like you correctly wrote.